### DISCUSSION TITLE
Update to Dutch locale [nl]

### DISCUSSION THREAD

**Discussion by calangh** [aclap-dev/video-downloadhelper#750] [Language & Translation]
I filled in the missing translations for dutch
[nl.txt](https://github.com/aclap-dev/video-downloadhelper/files/15474982/nl.txt)

**Comment by OBAK2** [comment]
Me too, filled all missing fields for Dutch... Here you go guys!
[nl.txt](https://github.com/aclap-dev/video-downloadhelper/files/15476930/nl.txt)

**Reply by pbdown** [reply]
Also filled in missing fields
[nl.txt](https://github.com/aclap-dev/video-downloadhelper/files/15485798/nl.txt)

**Comment by avda-nl** [comment]
And some more...

[nl.txt](https://github.com/aclap-dev/video-downloadhelper/files/15497734/nl.txt)

**Comment by paulrouget** [comment]
Thank you! These will all be in v9.1.
